packages/circuits/src/UnirepCircuit.ts

```diff
@@ -0,0 +1,157 @@
+import path from 'path'
```

Comment:
This module will have trouble with browser support.

Reply:
yeah that makes sense